pom.xml

```diff
@@ -101,6 +101,11 @@
             <artifactId>killbill-platform-osgi-bundles-lib-killbill</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.kill-bill.billing.plugin</groupId>
+            <artifactId>killbill-plugin-api-invoice</artifactId>
+            <scope>provided</scope>
+        </dependency>
         <dependency>
```

Comment:
Is this dependency intentionally deleted?

Reply:
Hi Reshma,

It is needed as we need to use PluginInvoicePluginApi class for HelloWorldInvoicePluginApi

Reply:
Ahh sorry, I thought you had deleted this dependency 😀